core/src/main/java/com/github/kaktushose/jda/commands/internal/Helpers.java

```diff
@@ -111,6 +114,23 @@ public static boolean checkSignature(MethodDescription method, Collection<Class<
         return false;
     }
 
+    public static void checkDetached(IDetachableEntity entity, Class<?> origin) {
```

Comment:
Since we have the same logic for ReplyConfig, we could combine them to one method perhabs?